app/Dockerfile

```diff
@@ -1,18 +1,13 @@
-# Use the Active LTS version of Node.
-# See https://nodejs.org/en/about/releases/
-FROM node:16-alpine
-# Keep container packages up-to-date.
-# -U runs both apk update and apk upgrade.
-RUN apk -U upgrade
-# Copy just the package data to the working directory.
-COPY yarn.lock /srv
-COPY package.json /srv
-WORKDIR /srv
-# Install application dependencies.
-RUN yarn install --frozen-lockfile
-# Copy all the remaining application files (ignoring files in .dockerignore) to the working directory.
-COPY . /srv
-# Build the application.
-RUN yarn build
-# Run the application.
-CMD yarn start
+# Simple hello, world image mostly copied
+# from https://github.com/crccheck/docker-hello-world
```

Comment:
I had to dig a little into the reference repo to understand how to run/test it locally — might be nice to add more detail here or in contributor documentation about how to verify the release infrastructure is working.

e.g. running `make release-build`, `docker run -p 80:8000 template-infra-app`, and visiting `localhost`

<img width="498" alt="Screen Shot 2022-09-30 at 11 02 28 AM" src="https://user-images.githubusercontent.com/2308368/193299286-5d6d874f-b4bb-4d66-9ed0-9a20ef7ee444.png">

Reply:
Sounds good, added to this file. I'm struggling with where to put some documentation that's specific to contributors to the template that the project teams won't want to keep. I wonder if we should have something like `docs/template-only/*` folder so that project teams can do `cp -r template-infra/docs docs && rm -fr docs/template-only`

Reply:
oh gotcha, that's a good point. For this scenario, keeping it in this file seems okay, you could also have it in `app/README.md` since the app/ folder gets replaced anyways.